### DISCUSSION TITLE
Allow setting a database password for the local Supabase environment

### DISCUSSION THREAD

**Discussion by timobehrens** [supabase/supabase#21216] [Feature Requests]
I am using the local Supabase dev environment to develop an app. I started integrating PowerSync and they provide [instructions](https://docs.powersync.com/integration-guides/supabase-+-powersync/local-development) on how to connect PowerSync (cloud hosted) with a local Supabase project via ngrok. Since the default password for the database is "postgres", my local environment is now open to the internet with the default DB password. Currently it is not possible to change the Supabase DB password, leaving the DB exposed.

**Comment by sosensible** [comment]
Fundamentally, you should not have sensitive data in a dev database. After that consideration, if you are concerned with the security on NGrok, you shouldn't do this.
